packages/openapi-generator/src/cli.ts

```diff
@@ -41,9 +41,30 @@ export = class GenerateOpenApiClient extends Command {
     }),
     clearOutputDir: flags.boolean({
       name: 'clearOutputDir',
-      description: 'Remove all files in the ouput directory before generation.',
+      description:
+        'Remove all files in the output directory before generation.',
       default: false,
       required: false
+    }),
+    versionInPackageJson: flags.string({
+      name: 'versionInPackageJson',
+      description:
+        'By default, when generating package.json file, the generator will set a version by using the generator version. It can also be set to a specific version.',
+      required: false
+    }),
+    generatePackageJson: flags.boolean({
```

Comment:
Make a compatibility note for all files which are created now and were not created in the past.